packages/block-editor/src/components/inserter/style.scss

```diff
@@ -299,7 +305,7 @@ $block-inserter-tabs-height: 44px;
 	@include break-medium {
 		border-left: $border-width solid $gray-200;
 		padding: 0;
-		left: 100%;
+		left: $secondary-sidebar-width;
```

Comment:
This is now a fixed number, but we need it to be to know how far to offset. The category panel content still needs to be absolutely positioned as it's within the flow of the tabs.